README.md

```diff
@@ -52,9 +52,7 @@ Every PR must be prepared using the following commands:
 ```bash
 cd $GOPATH/src/github.com/go-clang/gen
 scripts/switch-clang-version.sh 3.4
-make install
-make test
-make lint
+make pr
```

Comment:
Then i'm not a fan of this naming tbh.
Though I need this command for a pull request I won't exclusively use it for that and pr always has (for me at least) something to do with a version control system adding, staging, commiting etc.